### ISSUE TITLE
When setting a border radius blur disappears.

### ISSUE THREAD

**Issue by grundmanise** [react-native-community/react-native-blur#161]
Hi, first of all thank you for the component.

I can't set a topLeft & topRight radius on the `BlurView`.
When I am setting `borderTopLeftRadius`/`borderTopRightRadius`/... on the `BlurView` blur disappears.
Setting it on the wrapper `View` - same issue.

**Comment by Kureev** [comment]
Hi @grundmanise, can you provide us some code so we can reproduce your issue?

**Comment by grundmanise** [comment]
Yes, sure.

```
<BlurView style={styles.blurView} blurType="light" blurAmount={15} />

const styles = StyleSheet.create({
    blurView: {
        width: 200,
        height: 200,
        backgroundColor: 'rgba(0,0,0,.1)',
        borderTopLeftRadius: 100,
        borderTopRightRadius: 100,
        // borderRadius: 100, <- works
    },
});
```

```
<View style={styles.wrapper>
   <BlurView style={styles.blurView} blurType="light" blurAmount={15} />
</View>

const styles = StyleSheet.create({
    blurView: {
        flex: 1,
        backgroundColor: 'rgba(0,0,0,.1)',
    },
   wrapper: {
        width: 200,
        height: 200,
        borderTopRightRadius: 100,
        borderTopLeftRadius: 100,
        overflow: 'hidden' //<- with overflow: 'visible' BlurView does not disappear, but borderRadius is not visible.
    },
});
```

**Comment by grundmanise** [comment]
I think this could relate to https://github.com/react-native-community/react-native-blur/issues/154

**Comment by Kureev** [comment]
So `borderRadius` works, but if you use one of the `border<Corner>Radius` it breaks? If so, that sounds to me like a RN layout issue.

Actually, it's quite strange that `rgba` as a background color works for you. IIRC, UIBlurView can't have any opacity.

**Comment by grundmanise** [comment]
> So `borderRadius `works, but if you use one of the `border<Corner>Radius` it breaks


Exactly. 

Can `_visualEffectView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;` , `dispatch_async` and not subclassing `RCTView` cause this issue?

Should have mentioned: I only tried on iOS (device).

**Comment by robsquires** [comment]
Hiya folks, I'm having the same issue (`iPhone 6|ios 10.2.1|RN 0.40.0`)
Would like to try + fix but I'm not particularly experienced with iOS. Any pointers that might help my investigation?

**Comment by radetsky** [comment]
Same issue.

**Comment by ejkkan** [comment]
Anyone got this working?

**Comment by radetsky** [comment]
Yes, it's working. Read https://github.com/react-native-community/react-native-blur/issues/161#issuecomment-286898330 and you will see that with a wrapper with overflow property it just works.

**Comment by C-odes** [comment]
Hey guys. In my case,  borderRadius has no effect. 
: 
![image](https://user-images.githubusercontent.com/42934761/126865239-4ca328dc-9dde-437c-8e99-0db2e8b197bd.png)

Code: 

`<BlurView
    style={{ borderRadius:20}}
    overlayColor={'transparent'}
    blurType="light"
    blurAmount={10}
>`
